### PR TITLE
libssh2 1.5.0

### PR DESCRIPTION
**Pull request by DomT4**
Release to fix [CVE-2015-1782](http://www.libssh2.org/adv_20150311.html), amongst a hoard of less important bug fixes.

Added a test. It works okay, and the documentation suggests it’s right, but my hatred of C is well known so I’m not attempting to vouch as to whether this test sucks or not.

### CODE REVIEW

**Comment by DomT4** [comment]
@xu-cheng Feel free to review/judge/laugh at my C test here :)

File: Library/Formula/libssh2.rb [hunk 1]
```diff
@@ -45,4 +44,18 @@ def install
...
   end
+
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+     #include <libssh2.h>
+
+     int main(void)
+     {
+      libssh2_exit();
+     }
```
Comment (xu-cheng) [inline Library/Formula/libssh2.rb]: Maybe a `return 0;` here.

Comment (DomT4) [inline Library/Formula/libssh2.rb]: Added, Thanks.

File: Library/Formula/libssh2.rb [hunk 2]
```diff
@@ -45,4 +44,19 @@ def install
...
+  test do
+    (testpath/"test.c").write <<-EOS.undent
+     #include <libssh2.h>
+
+     int main(void)
+     {
+      libssh2_exit();
+      return 0;
+     }
+    EOS
```
Comment (tdsmith) [inline Library/Formula/libssh2.rb]: :dancer: for test; fixed up the heredoc's indent to two characters

Comment (DomT4) [inline Library/Formula/libssh2.rb]: Thanks Tim! Will try to ensure I hit the two-step indent inside testfiles in future :).

**Pull request closed without merging**